tests/buildifier/BUILD

```diff
@@ -0,0 +1,12 @@
+sh_test(
+    name = "buildifier_test",
+    srcs = ["buildifier_test.sh"],
+    size = "medium",
+    data = [
+        "//tests/unittest",
+        "@bazel_tools//tools/bash/runfiles",
+        "//:WORKSPACE",
```

Comment:
Can you explain why you are exporting the `WORKSPACE` and including it in this test? If it is to create a workspace for the test, it is better to create a new `WORKSPACE` that refers to the parent using [local_repository](https://bazel.build/rules/lib/repo/local#local_repository).

Reply:
@cgrindel sorry for the delay, if you look at runner.bat.template line 24, we need to find the absolute path to the workspace directory, and there is no good way in bat/bazel to do this other than looking at the target of a symlink. Exporting and depending on WORKSPACE gives us this symlink. Any other file in root folder of workspace would also be fine.

Reply:
Are you trying to find the root of a test workspace? I would expect a buildifier test to create a workspace, execute buildifier, and then verify the changes. I don't believe that any of that requires access to this repository's `WORKSPACE` file.

Reply:
buildifier_test already has sandbox escape mechanism, implemented as the 'sandbox' and 'workspace' attributes to buildifier_test macro. This is documented here: https://github.com/keith/buildifier-prebuilt/blob/32e63ed4a91332fb26e6ee76c168eab437764cc2/buildifier/factory.bzl#L69, and is one of the things that this PR fixes on windows.

Reply:
Perhaps, I am not familiar enough with this use case. I will defer to @keith.